ilastik/workflows/neuralNetwork/_localWorkflow.py

```diff
@@ -0,0 +1,248 @@
+import argparse
+import logging
+
+import numpy
+
+from ilastik.config import cfg
+from ilastik.workflow import Workflow
+from ilastik.applets.dataSelection import DataSelectionApplet
+from ilastik.applets.serverConfiguration.types import ServerConfig, Device
+from ilastik.applets.neuralNetwork import NNClassApplet, NNClassificationDataExportApplet
+from ilastik.applets.batchProcessing import BatchProcessingApplet
+from ._localLauncher import LocalServerLauncher
+
+from lazyflow.operators import tiktorch
+
+from lazyflow.graph import Graph
+
+
+logger = logging.getLogger(__name__)
+
+
+class LocalWorkflow(Workflow):
```

Comment:
With the local workflow being more or less the same than the remote one, minus the serverconfig applet, shouldn't it make sense to subclass the remote from the local one, or at least having some common base?